packages/editor-ui/src/components/WorkflowEvaluation/EditEvaluation/TagsInput.vue

```diff
@@ -0,0 +1,98 @@
+<script setup lang="ts">
+import { useI18n } from '@/composables/useI18n';
+import type { ITag } from '@/Interface';
+import { createEventBus } from 'n8n-design-system';
+import { computed } from 'vue';
+
+export interface TagsInputProps {
+	modelValue?: {
+		isEditing: boolean;
+		appliedTagIds: string[];
+	};
+	allTags: ITag[];
+	tagsById: Record<string, ITag>;
+	isLoading: boolean;
+	startEditing: (field: string) => void;
+	saveChanges: (field: string) => void;
+	cancelEditing: (field: string) => void;
+}
+
+const props = withDefaults(defineProps<TagsInputProps>(), {
+	modelValue: () => ({
+		isEditing: false,
+		appliedTagIds: [],
+	}),
+});
+
+const emit = defineEmits<{ 'update:modelValue': [value: TagsInputProps['modelValue']] }>();
+
+const locale = useI18n();
+const tagsEventBus = createEventBus();
+const getTagName = computed(() => (tagId: string) => {
+	return props.tagsById[tagId]?.name ?? '';
+});
+
+function updateTags(tags: string[]) {
+	const newTags = tags[0] ? [tags[0]] : [];
+	emit('update:modelValue', {
+		...props.modelValue,
+		appliedTagIds: newTags,
+	});
+}
+</script>
+
+<template>
+	<div data-test-id="workflow-tags-field">
+		<n8n-input-label label="Tag name" :bold="false" size="small">
+			<div v-if="!modelValue.isEditing" :class="$style.tagsRead" @click="startEditing('tags')">
+				<n8n-text v-if="modelValue.appliedTagIds.length === 0" size="small">
+					{{ locale.baseText('workflowEvaluation.edit.selectTag') }}
+				</n8n-text>
+				<n8n-tag
+					v-for="tagId in modelValue.appliedTagIds"
+					:key="tagId"
+					:text="getTagName(tagId)"
+					data-test-id="evaluation-tag-field"
+				/>
+				<n8n-icon-button
+					:class="$style.editInputButton"
+					icon="pen"
+					type="tertiary"
+					size="small"
+					transparent
+				/>
+			</div>
+			<TagsDropdown
```

Comment:
Can we somehow adapt this TagsDropdown for a single tag select use case? It opens list of tags when you click on the input with selected tag, making a false impression that this is a multi-tag input.

Reply:
I'll do this in the next step — selecting executions and mocking node